soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6668,7 +6668,14 @@ s32 func_8083F0C8(Player* this, PlayState* play, u32 arg2) {
                 this->actor.world.pos.z = tempZ + (wallDistance * wallPolyNormZ);
                 func_80832224(this);
                 this->actor.prevPos = this->actor.world.pos;
-                func_80832264(play, this, &gPlayerAnim_link_child_tunnel_start);
+                if (CVarGetInteger("gCrawlSpeed", 0) < 2) {
+                    func_80832264(play, this, &gPlayerAnim_link_child_tunnel_start);
+                } else {
```

Comment:
if case 100 lines below looks good, I'd still recommend changing this one not only unify the logic, but also because you've still got some cvar gets that are using 0 as the default instead of 1.

Reply:
Okay, _now_ this is done. Appreciate the patience!